static/js/protein/page.tsx

```diff
@@ -111,15 +118,19 @@ export class Page extends React.Component<PagePropType, PageStateType> {
     Using the split we get the ProteinName and SpeciesName separately
     */
     const splitNodeName = this.props.nodeName.split("_");
+    const proteinLink = `${GRAPH_EXPLORER_REDIRECT}${this.props.dcid}`;
     const proteinDescription = getProteinDescription(this.state.data);
     return (
       <>
         <h2>{splitNodeName[0] + " (" + splitNodeName[1] + ")"}</h2>
+        <h6>
+          <a href={proteinLink}>Graph Explorer View</a>
```

Comment:
do you mean Graph Browser? If yes, also update the redirect variable name

Reply:
Yes, updated it!